src/assets/js/components/options/sign-in.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
```

Comment:
Directory name `options` could be better

Reply:
@iamplugged, a browser extension can be composed of an [options page](https://developer.chrome.com/docs/extensions/reference/manifest) and a [popup page](https://developer.chrome.com/docs/extensions/develop/ui/add-popup), so I decided to rename it to `options`.